README.md

```diff
@@ -1,2 +1,27 @@
 # kaoto-next
 Next version of the UI of the Kaoto project
+
+## Building the project
+This project leverages `vite`, you can find more information about it [here](https://vitejs.dev/config/)
+
+### Requirements
+- NodeJS (>v18.x)
+- yarn (>v3.x)
+
+### Steps
+1. Clone the repository
+2. In the recently cloned folder, run `yarn install` to install all the dependencies
+(By default, `@kaoto-next/camel-catalog` will be built as well, using the `mvn` wrapper)
```

Comment:
it doesn't seem to trigger it for me:
![image](https://github.com/KaotoIO/kaoto-next/assets/1105127/6ea802af-f7c9-486b-92be-ede4a0aae6dc)